inject.js

```diff
@@ -62,11 +63,15 @@ module.exports = function (name, override) {
 
   if (!result.connections.incoming) {
     result.connections.incoming = {
-      net: [{ host: result.host, port: result.port, scope: "public", "transform": "shs" }],
-      ws: [{ host: result.host, port: result.ws.port, scope: "device", "transform": "shs" }]
+      net: [{ host: result.host, port: result.port, scope: 'public', 'transform': 'shs' }],
+      ws: [{ host: result.host, port: result.ws.port, scope: 'device', 'transform': 'shs' }]
     }
   }
-  return result
-}
 
+  // *** LEGACY TIDYUP ***
```

Comment:
I dig this. I don't have a solid grasp on which modules use `config.port` and `config.host`, but this would require a major version bump and some release notes, yeah?

Reply:
`config.port` and `config.host` are useful configurations because you can set them with environment variables `ssb_port` and `ssb_host`. removing these means we can't configure pubs in a standard way. that being said, my plan for PeachCloud is to avoid using these parts of the jsbot stack altogether, but recent changes are currently breaking `ahdinosaur/ssb-pub`: https://github.com/ahdinosaur/ssb-pub/issues/15#issuecomment-449977926

Reply:
oops, my above comment is probably in the wrong place, deleting `config.host` from the returned object is good, not being able to set `ssb_host` to configure the external host is meh.

Reply:
Cool, so that sounds like a :+1: from @ahdinosaur which is good enough for me.
What I've got supports old inputs, but also removes a footgun!

Reply:
turns out this was needed by ssb-server/bin. It's not documented anywhere, sure. and even I had forgotten that was used. But there generally tend to be things like this. The best way to catch things like this is to symlink a module into it's dependants and then run those tests.

`cd ssb-server; cd node_modules; rm -rf ssb-keys; ln -s ../../ssb-keys; cd ..; npm test`
This is the last line of defence against releasing broken stuff, and it would make me _very happy_ if people making PRs would run this before they submit it for review.